bash_scripts/install_meltingpot.sh

```diff
@@ -20,4 +20,7 @@ pip install https://github.com/deepmind/lab2d/releases/download/release_candidat
 mkdir ../packages
 cd ../packages
 git clone -b main https://github.com/deepmind/meltingpot
-pip install meltingpot/
+# Temp - Pin version before python 3.9 requirement.
```

Comment:
Do you think it will only be temporary and that meltingpot will support older versions in the future? I also wonder for how long we should support all these? I.e. how common is it that other libraries do not support python 3.7 for example meaning that we would have to either not use them, or selectively support them based on the version of python you're using Mava with.

Reply:
I think meltingpot has said goodbye to older versions of python, not 100% sure why. For us, I think we should support them as long as they don't prove to be an obstacle (like python 3.6 was with that weird extra dependency we had to manually down).